tools/cli/help.txt

```diff
@@ -1,19 +1,25 @@
 >>>
 (the first line of each entry is the summary for autogenerated command lists)
-Usage: meteor [--release <release>] [--help] <command> [args]
-       meteor help <command>
+Usage: meteor [--release <release>] [--help] <built-in command> [args]
+       meteor help <built-in command>
+       meteor <external command> [args]       
        meteor [--version] [--arch]
+       
 
 With no arguments, 'meteor' runs the project in the current
 directory in local development mode. You can run it from the root
 directory of the project or from any subdirectory.
 
 Use 'meteor create <path>' to create a new Meteor project.
 
-Commands:
+Built-in Commands:
 {{commands}}
 
-See 'meteor help <command>' for details on a command.
+See 'meteor help <built-in command>' for details on a command.
+
+You can use 'meteor <external command> [args]' to run a binary command
+installed previously with 'meteor npm install -g <package with command>',
+which will store the package under the global meteor installation folder.
```

Comment:
> I still think that a more in-depth explanation about node, npm and included tools (python, mongo, etc...) could be added to the guide too,

Absolutely!  http://docs.meteor.com/commandline.html#meteornpm could certainly be expanded to explain that `--global` has purpose!  I think `jsdoc` is super reasonable too.  Feel free to open issues/PRs with docs as appropriate! :)

> that if we are looking for the place were meteor stores them, is it in the dev_bundle folder.

The `dev_bundle` is certainly magical for most users; the fact that of `meteor npm --global` is quite different than `npm --global` is probably worth making more clear.